frontend/src/views/critical-ratio.js

```diff
@@ -0,0 +1,115 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import plane from "../images/plane.png";
+import { useRound } from "../selectors";
+import { useAddCriticalRatio } from "../action-hooks";
+import { Button } from "../components/ui";
+
+function CriticalRatio() {
+  const [criticalRatio, setCriticalRatio] = React.useState("");
+  const [isEnabled, setIsEnabled] = React.useState(false);
+
+  const addCriticalRatio = useAddCriticalRatio();
+
+  const state = useRound();
+
+  const buttoncomponent = {
```

Comment:
I suggest using camelCase to all variables' name.

Reply:
yeah, you're right.